cmake/CompilerOptions.cmake

```diff
@@ -58,5 +58,5 @@ elseif("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /w")
     endif()
 else()
-    message(FATAL_ERROR "Unknown compiler: ${CMAKE_CXX_COMPILER_ID}")
+    message(WARNING "Unknown compiler, not able to set compiler options for ${CMAKE_CXX_COMPILER_ID}")
```

Comment:
Is `CMAKE_CXX_COMPILER_ID` a readable variable or do we need to add something like:
`message(WARNING "Unknown compiler, not able to set compiler options for compiler ID: ${CMAKE_CXX_COMPILER_ID}")`

Reply:
What's a readable variable?

Reply:
If the ID is `12345` or `a1fa335` I would say it is unreadable, but if it is `gcc--9.1.2` or `clang-10.1.2` I would say it is readable.

Reply:
Ah, now I understand what you mean. Good point! It's "Clang", "GNU" and "MSVC" for those three compilers. I assume others are similarly readable.